app/views/users/new.erb

```diff
@@ -0,0 +1,26 @@
+<form action="/users/new" method="post" id="signup-form">
+  <fieldset>
+    <legend>Please sign up to use chitter</legend>
+    <p>
```

Comment:
Nice use of `<p>` tags to break up each element, this is what I would consider the best way of marking up a form :)